agent/config/agent.yaml

```diff
@@ -0,0 +1,26 @@
+# agent-integrations-next.yaml is intended to be used with
+# docker-compose.integrations-next.yaml. It demonstrates a config with
+# --enable-features=integrations-next and --config.expand-env, but does not
+# test absolutely all integrations.
```

Comment:
These are things that will be added later?

Reply:
Nope, this is just there from the template I was using. I'll remove it to avoid confusion